docs/api.md

```diff
@@ -4041,6 +4042,8 @@ const browser = await chromium.launch({  // Or 'firefox' or 'webkit'.
     - `bypass` <[string]> Optional coma-separated domains to bypass proxy, for example `".com, chromium.org, .domain.com"`.
     - `username` <[string]> Optional username to use if HTTP proxy requires authentication.
     - `password` <[string]> Optional password to use if HTTP proxy requires authentication.
+  - `acceptDownloads` <[boolean]> Whether to automatically download all the attachments. Defaults to `false` where all the downloads are canceled.
+  - `downloadsPath` <[string]> If specified, accepted downloads are downloaded into this folder. Otherwise, temporary folder is created and is deleted when browser is closed.
```

Comment:
Add a note that it only makes sense when `acceptDownloads` is set?

Reply:
This folder is created regardless the `acceptDownloads`, so I'd like to be careful about the wording here. I deliberately put them next to each other and say "accepted downloads" here.